frontend/pages/dataset/_id/annotation-mode/index.vue

```diff
@@ -57,25 +57,16 @@ export default {
     };
   },
   beforeRouteLeave(to, from, next) {
-    if (!this.areResponsesUntouched) {
-      let message = "";
-      switch (to.name) {
-        case "datasets":
-        case "dataset-id-settings":
-        case "user-settings":
-          message = "Your changes will be lost if you leave the current page";
-          break;
-        default:
-        // do nothing
-      }
-      message.length &&
-        this.showNotification({
-          eventToFireOnClick: async () => {
-            await next();
-          },
-          message,
-          buttonMessage: this.buttonMessage,
-        });
+    const isNotificationForThisRoute =
+      !this.areResponsesUntouched &&
+      ["datasets", "dataseat-id-settings", "user-settings"].includes(to.name);
```

Comment:
This typo error breaks the dataset settings prevention. Please, apply your changes carefully and be sure that are working before requesting a review.

```suggestion
      ["datasets", "dataset-id-settings", "user-settings"].includes(to.name);
```